site/content/community/_index.html

```diff
@@ -35,11 +35,11 @@ <h3><a href="{{< param "slack_url" >}}">Chat with us on our Slack channel</a></h
             </div>
             <div class="content">
                 <h3><a href="{{< param "community_url" >}}">Community Meetings</a></h3>
-                <p>Please join us during our online <a href="https://go.pinniped.dev/community/agenda">Community Meetings</a>, occurring every first and third Thursday of the month at 9 AM PT / 12 PM ET.</p>
-                <p>Use <a href="https://go.pinniped.dev/community/zoom">this Zoom link</a> to attend and add any agenda items you wish to discuss to <a href="https://go.pinniped.dev/community/agenda">the notes document</a>.</p>
-                <p>Join our <a href="https://groups.google.com/u/1/g/project-pinniped">Google Group</a> to receive invites to the meeting</p>
+<!--                <p>Please join us during our online <a href="https://go.pinniped.dev/community/agenda">Community Meetings</a>, occurring every first and third Thursday of the month at 9 AM PT / 12 PM ET.</p>-->
+<!--                <p>Use <a href="https://go.pinniped.dev/community/zoom">this Zoom link</a> to attend and add any agenda items you wish to discuss to <a href="https://go.pinniped.dev/community/agenda">the notes document</a>.</p>-->
+                <p>Join our <a href="https://groups.google.com/u/1/g/project-pinniped">Google Group</a> to receive invites to community meetings</p>
```

Comment:
Add the same blurb about joining the group for updates, release meetings etc here.

Reply:
Thanks, updated the original commit on this PR to include this.